yashsachdev.HabitTracker/DatabaseClass.cs

```diff
@@ -0,0 +1,59 @@
+﻿public static class DatabaseClass
+{
+    public static readonly string connectionString = "Data Source=Habit-Tracker.db";
+    public static void CreateDatabase()
+    {
+        using (SqliteConnection connection = new SqliteConnection(connectionString))
+        {
+            try
+            {
+                if (connection.State != ConnectionState.Open)
+                {
+                    connection.Open();
+                }
+                Console.WriteLine(connection.State.ToString());
+                SqliteCommand command = new SqliteCommand($"ATTACH DATABASE 'Habit-tracker.db' AS Habit;", connection);
+                command.ExecuteNonQuery();
+                Console.WriteLine("Database Created");
+            }
+            catch (SqliteException ex)
+            {
+                Console.WriteLine("error:" + ex.Message);
+            }
+            finally
+            {
+                connection.Close();
+            }
+        }
+    }
+    public static void CreateTable()
+    {
+        try
+        {
+            using (SqliteConnection connection = new SqliteConnection(connectionString))
+            {
+                connection.Open();
+                using (SqliteCommand command = new SqliteCommand())
+                {
+                    command.Connection = connection;
+                    StringBuilder sb = new StringBuilder();
+                    sb.Append("CREATE TABLE IF NOT EXISTS User (User_Id INTEGER PRIMARY KEY AUTOINCREMENT,Name VARCHAR(45) NOT NULL,Email VARCHAR(45) NOT NULL,Password VARCHAR(8) NOT NULL);");
```

Comment:
![image](https://user-images.githubusercontent.com/108711476/220470897-b2e751ab-fd41-4232-b946-4c14b2608760.png)

This looks very spaghetti-y. Use a verbatim string such as this: 

![image](https://user-images.githubusercontent.com/108711476/220471066-5eae1f42-f0c5-4195-b642-3c7bf49a9abd.png)